### ISSUE TITLE
`weave --local create-bridge` fails when docker isn't running

### ISSUE THREAD

**Issue by rade** [weaveworks/weave#1552]
```
$ weave --local create-bridge
Post http:///var/run/docker.sock/v1.20/containers/create: dial unix /var/run/docker.sock: no such file or directory.
* Are you trying to connect to a TLS-enabled daemon without TLS?
* Is your docker daemon up and running?
```

This used to work and is important for folks that want to tell docker to use the weave bridge. Does that even work in principle with fastdp? I'm guessing not, in which case we should perhaps spit out an error saying so, with advice to set `WEAVE_NO_FASTDP` (which makes the above work). May turn out to be more trouble than it's worth, in which case this becomes just a "backward compatibility breaking change" for the release notes.

**Comment by rade** [comment]
@dpw @awh thoughts?

**Comment by dpw** [comment]
I'll deal with it later today.

**Comment by dpw** [comment]
I have the "attach docker0 to the weave network" thing I mentioned working.  It needs a bit more polish, so I guess it's too late for 1.2.0.  But it's simple enough that it seems like potential 1.2.1 material if we get any feedback on this.

**Comment by rade** [comment]
Closed by #1562.
